### ISSUE TITLE
Document block color hex notation

### ISSUE THREAD

**Issue by AnmAtAnm** [google/blockly#917]
https://developers.google.com/blockly/guides/create-custom-blocks/define-blocks where it mentions the HSV model.

**Comment by NeilFraser** [comment]
Keep in mind that not documenting this (other than in JSDoc) was a _very_ deliberate decision.

When Blockly started we did a survey of other block-based programming systems.  At the time there were quite a lot of them.  One striking feature common to most was horrible colour schemes.  For whatever reason, giving developers a free choice in colour selection was a recipe for an [angry fruit salad](http://www.catb.org/jargon/html/A/angry-fruit-salad.html).  We got a UX consultant to help us with Blockly's colour selection and observed that they were (without knowing it) using the HSV model to obtain harmonious colours.  So that became Blockly's colour scheme.

If the common RGB model is made available to developers they will choose this over the more obscure HSV model.  And we'll be right back to the days of day-glo blocks.  We'll also lose a lot of visual consistency between block-based projects, which is one of the goals of the Kids Coding effort.

I highly recommend leaving RGB undocumented.  This is a feature, not a bug.
